src/components/bluebutton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, label, ...props }) =>
+(
+  <div
+    className={styles.our_button}
+    role="button"
+    tabIndex="-1"
+    {...props}
+    onClick={(event) => {
+      event.preventDefault();
+      targetTo();
+    }}
+  >
+    {label}
+  </div>
+  );
+
+Button.propTypes = {
+  targetTo: PropTypes.func.isRequired,
+  label: PropTypes.string.isRequired,
+};
+
+Button.defaultProps = {
+  label: 'Name me!',
+};
+
+module.exports = Button;
+
+// todo: Add Flex box integration
+//       Add SASS implemenation
+//       Add additional abstractions
+//       Look into containers or stand alone
```

Comment:
I think some of these Todo's are outdated.
They should also all be of the format `// TODO: <text>` so that [automated tools](https://www.npmjs.com/package/code-notes) can find them.